projects/storefrontstyles/scss/layout/page-templates/_myaccount-v2-page.scss

```diff
@@ -1,33 +1,19 @@
 %MyAccountViewPageTemplate {
   max-width: var(--cx-page-width-max);
   margin: auto;
-  display: flex;
   cx-page-slot {
     cx-banner {
-      height: 120px;
-      margin-top: 40px;
-      margin-bottom: -500px;
-      border: 1px solid var(--cx-color-light);
+      margin: 2rem 0rem 0.5rem 0rem;
+      border: 1px solid var(--cx-color-medium);
     }
   }
-
-  @include media-breakpoint-down(md) {
-    flex-direction: column;
-  }
   .LeftContentSlot {
+    float: left;
     max-width: 25%;
-    @include media-breakpoint-down(md) {
-      width: 100%;
-      max-width: none;
-    }
   }
 
   .RightContentSlot {
+    float: left;
```

Comment:
[NITPICK] 
Do we _really_ have to use `float` instead of `flex` for styling `%MyAccountViewPageTemplate`? I believe for consistency with the rest of the project, it would be good to stick to `flex` styling.

I can see `float` is used also in other files in my-account-v2 area. Perhaps it would be good to fix it before 6.6 release:
![image](https://github.com/SAP/spartacus/assets/4001059/4b0ee5d4-0b23-4e18-9b00-60dfdf583919)
(yes, I can see `float` is used also in ASM, which ideally should have used `flex`)